### PR TITLE
Adding more modern real world project examples.

### PR DESCRIPTION
**Pull request by okrasz**
As we discussed on GWT.create - I've added some more modern projects using GWT. For Google I changed it from Moderator to Inbox and also added a list of Google services that are using GWT, that I knew of. I didn't remove other projects.

### CODE REVIEW

**Comment by okrasz** [comment]
I've signed the CLA.

Best regards,
Marcin Okraszewski

File: src/main/markdown/examples.md [hunk 1]
```diff
@@ -144,13 +144,51 @@ GWT is being used by tens of thousands of projects around the world. Take a look
...
+
+<div class="project">
+  <div class="screenshot">
+    <a href="https://ruxit.com" target="_blank"><img src="images/ruxit.png" alt="ruxit"/></a>
+  </div>
+  <div class="info">
+    <div class="name"><a href="https://ruxit.com" target="_blank">ruxit</a></div>
+    <div class="snippet">ruxit is a cloud-based application performance management solution that offers unified monitoring of 
+user experience, servers, applications, infrastructure and networks &ndash; all with simple auto-configuration. The entire product UI 
+was built using GWT. Use the free trial to see it in action.</div>
```
Comment (jDramaix) [inline src/main/markdown/examples.md]: *free trial*
If you have a direct link, add it

File: src/main/markdown/examples.md [hunk 2]
```diff
@@ -144,13 +144,51 @@ GWT is being used by tens of thousands of projects around the world. Take a look
...
+		<a href="https://www.google.com/webmasters/tools/home" target="_blank">Webmaster Tools</a>,
+		<a href="http://www.google.com/moderator/" target="_blank">Moderator</a>,
+		<a href="http://www.google.com/blogger" target="_blank">Blooger&nbsp;Admin</a>,
+		<a href="https://www.google.com/wallet/" target="_blank">Wallet</a>,
+		<a href="http://www.google.com/webfonts" target="_blank">WebFonts</a>,
+		<a href="http://www.google.com/script/" target="_blank">Apps&nbsp;Script</a>,
+		<a href="https://www.google.com/producer" target="_blank">Currents&nbsp;Producer</a>,
+		<a href="http://www.google.com/flights" target="_blank">Flights</a>,
+		<a href="https://www.google.com/fusiontables/DataSource?dsrcid=2049253" target="_blank">Fusion&nbsp;Tables</a>,
+		<a href="http://www.google.com/hotelfinder" target="_blank">Hotel&nbsp;Finder</a>
```
Comment (jDramaix) [inline src/main/markdown/examples.md]: @dankurka Could you review this part concerning Google ?

**Comment by jDramaix** [comment]
Thanks for this contribution

File: src/main/markdown/examples.md [hunk 3]
```diff
@@ -144,13 +144,51 @@ GWT is being used by tens of thousands of projects around the world. Take a look
...
+    <div class="name"><a href="http://www.google.com/inbox" target="_blank">Google Inbox</a></div>
+    <div class="snippet">Google Inbox is a fresh start that goes beyond email to help you get back to what matters. 
+	It is the first Google service that is using the new approach to delivering multiplatform native applications using common code base. GWT is used to deliver the Web version.
+	<p>Google is using GWT in number of other public services, including:
+		<a href="https://docs.google.com/spreadsheets" target="_blank">Spreadsheets</a>,
+		<a href="http://www.google.com/groups" target="_blank">Groups</a>,
+		<a href="http://adwords.google.com " target="_blank">AdWords</a>,
+		<a href="https://www.google.com/webmasters/tools/home" target="_blank">Webmaster Tools</a>,
+		<a href="http://www.google.com/moderator/" target="_blank">Moderator</a>,
+		<a href="http://www.google.com/blogger" target="_blank">Blooger&nbsp;Admin</a>,
```
Comment (tbroyer) [inline src/main/markdown/examples.md]: Typo: Blogger, not Blooger

File: src/main/markdown/examples.md [hunk 4]
```diff
@@ -144,13 +144,51 @@ GWT is being used by tens of thousands of projects around the world. Take a look
...
-    <div class="name"><a href="http://www.google.com/moderator" target="_blank">Google Moderator</a></div>
-    <div class="snippet">Google Moderator is a tool that allows distributed communities to submit and vote on questions for talks, 
-presentations and events. Suggested questions as well as user votes are seen in real-time. Google Moderator's front-end is built
-using GWT and runs on top of Google's App Engine.</div>
+    <div class="name"><a href="http://www.google.com/inbox" target="_blank">Google Inbox</a></div>
+    <div class="snippet">Google Inbox is a fresh start that goes beyond email to help you get back to what matters. 
+	It is the first Google service that is using the new approach to delivering multiplatform native applications using common code base. GWT is used to deliver the Web version.
+	<p>Google is using GWT in number of other public services, including:
+		<a href="https://docs.google.com/spreadsheets" target="_blank">Spreadsheets</a>,
+		<a href="http://www.google.com/groups" target="_blank">Groups</a>,
```
Comment (tbroyer) [inline src/main/markdown/examples.md]: https://groups.google.com/forum/ (or https://groups.google.com/ ), but please use HTTPS

File: src/main/markdown/examples.md [hunk 5]
```diff
@@ -144,13 +144,51 @@ GWT is being used by tens of thousands of projects around the world. Take a look
...
+		<a href="http://adwords.google.com " target="_blank">AdWords</a>,
```
Comment (tbroyer) [inline src/main/markdown/examples.md]: HTTPS (should be true for all links below too)

File: src/main/markdown/examples.md (hunk 1 above)
Comment (okrasz) [inline src/main/markdown/examples.md]: Unfortunately not. It seems there is an inline form on the main page only.

**Comment by okrasz** [comment]
I've changed links to use HTTPS and fixed Blogger typo as @tbroyer suggested.
I've also improved a bit on the Inbox description, to reflect more the actual state (Spreadsheets was the first "hybrid" application) + added the link to Ray talking on this approach.

**Comment by jDramaix** [comment]
Thanks !

**Pull request merged**